plugins/woocommerce-admin/client/marketplace/components/feedback-modal/feedback-modal.tsx

```diff
@@ -0,0 +1,244 @@
+								stroke="#fff"
+							/>
+							<line
+								className="l"
+								x1="6.7669"
+								x2="22.323"
+								y1="4.4875"
+								y2="4.4875"
+								stroke="#fff"
+							/>
+						</svg>
+					</>
+				),
+				explicitDismiss: true,
+				onDismiss: incrementDismissedTimes,
+				actions: [
+					{
+						onClick: openModal,
+						label: 'Give feedback',
+					},
+				],
+			}
+		);
+	}
+
+	// eslint-disable-next-line react-hooks/exhaustive-deps -- [] => we only want this effect to run once, on first render
+	useEffect( maybeShowSnackbar, [] );
+
+	// We don't want the "How easy was it to find an extension?" dialog to appear forever:
+	const FEEDBACK_DIALOG_CAN_APPEAR =
+		new Date( SUPPRESS_IF_AFTER_DATE ) > new Date();
+	if ( ! FEEDBACK_DIALOG_CAN_APPEAR ) {
+		return <></>;
+	}
+
+	function easyToFindChanged( value: number ) {
+		setEasyToFindValidiationFailed( false );
+		setEasyToFind( value );
+	}
+
+	function meetsMyNeedsChanged( value: number ) {
+		setMeetsMyNeedsValidiationFailed( false );
+		setMeetsMyNeeds( value );
+	}
+
+	function submit() {
+		// Validate:
+		if ( easyToFind === 0 || meetsMyNeeds === 0 ) {
+			if ( easyToFind === 0 ) setEasyToFindValidiationFailed( true );
+			if ( meetsMyNeeds === 0 ) setMeetsMyNeedsValidiationFailed( true );
+			return;
+		}
+		// Send event to CES:
+		recordEvent( 'ces_feedback', {
+			action: CUSTOMER_EFFORT_SCORE_ACTION,
+			score: easyToFind,
+			score_second_question: meetsMyNeeds,
+			score_combined: easyToFind + meetsMyNeeds,
+			comments: thoughts,
+		} );
+		// Close the modal:
+		setOpen( false );
+		// Ensure we don't ask for feedback again:
+		dismissForever();
+	}
+
+	return (
+		<>
+			{ isOpen && (
+				<Modal
+					title={ __(
+						'How easy was it to find an extension?',
+						'woocommerce'
+					) }
+					onRequestClose={ closeModal }
+					className="woocommerce-marketplace__feedback-modal"
+				>
+					<p>
+						{ __(
+							'Your feedback will help us create a better experience for people like you! Please tell us to what extent you agree or disagree with the statements below.',
+							'woocommerce'
+						) }
+					</p>
+					<LikertScale
+						fieldName="extension_screen_easy_to_find"
+						title={ __(
+							'It was easy to find an extension',
+							'woocommerce'
+						) }
+						onValueChange={ easyToFindChanged }
+						validationFailed={ easyToFindValidiationFailed }
+					/>
+					<LikertScale
+						fieldName="extension_screen_meets_my_needs"
+						title={ __(
+							'The Extensions screen’s functionality meets my needs',
+							'woocommerce'
+						) }
+						onValueChange={ meetsMyNeedsChanged }
+						validationFailed={ meetsMyNeedsValidiationFailed }
+					/>
+					<TextareaControl
+						label={ __( 'Additional thoughts', 'woocommerce' ) }
+						value={ thoughts }
+						onChange={ ( value: string ) =>
+							setThoughts( value )
+						}
```

Comment:
```suggestion
						onChange={ ( value: string ) => setThoughts( value ) }
```

Good to have it in a single line?

Reply:
Fixed in [`216e306` (#39609)](https://github.com/woocommerce/woocommerce/pull/39609/commits/216e3061787923e070878c1491098fb984f53f47).